tenseal/sealapi.cpp

```diff
@@ -0,0 +1,101 @@
+#include "sealapi.h"
+
+#include <pybind11/operators.h>
+#include <pybind11/pybind11.h>
+#include <seal/seal.h>
+
+using namespace seal;
+using namespace std;
+namespace py = pybind11;
+
+void loadSEALAPI(py::module &m) {
```

Comment:
seal_api should be totally independent, it can be done like this
```suggestion
PYBIND11_MODULE(_tenseal_cpp, m) {
    py::module seal_api = m.def_submodule("seal");
    // "seal/biguint.h"
    py::class_<BigUInt>(seal_api, "BigUInt")
```
This way, you won't need to call it from binding.cpp